components/nimbus/src/tests/test_persistence.rs

```diff
@@ -0,0 +1,784 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO: Add unit tests
+// Possibly by using a trait for persistence and mocking it to test the persistence.
```

Comment:
I think we can remove those top comments :P I think they were added WAAAAY before, and now we have good test coverage on our persistence 🚀 but we never got rid of them 😆

Reply:
I was considering going through and doing a sweep of all of the todo's in these files 😂 (there's a lot)

Reply:
I'll take these out tho